docs/adminguide/frozen_users.rst

```diff
@@ -0,0 +1,114 @@
+.. Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
+
+.. _doc_adminguide_frozen_users:
+
+
+Frozen users
+============
+
+It is possible for server administrators to freeze specific Parsec users.
+
+This feature is not part of the Parsec revocation system, but a way for system administrators to temporarily block users from connecting to the server while waiting for organization administrators to actually revoke them. It is exposed as an HTTP route that requires the administration token. This way, it is possible for existing directory services to automatically freeze users that have been removed from the directory.
+
+HTTP ``users`` route
+--------------------
+
+This route is made available as ``/administration/organizations/<raw_organization_id>/users`` and requires an administration token.
+
+It only supports the ``GET`` method which lists information for all users, including Parsec ID, user name, user email, and frozen status.
+
+Here's an example using ``curl`` and ``jq``:
+
+  .. code-block:: bash
+
+    $ curl https://$server/administration/organizations/$organization/users \
+        -H "Authorization: Bearer $administration_token" | jq
+
+A successful request returns a JSON dictionary similar to the one below:
```

Comment:
I think it's _object_ instead of _dictionary_ in JSON terminology.
```suggestion
A successful request returns a JSON object with the following structure:
```

Reply:
[70d6ef2](https://github.com/Scille/parsec-cloud/pull/5842/commits/70d6ef29491522ed83c28a1dcd9f084b78896ef2)